content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -35,7 +35,7 @@ secretGenerator:
   - password=1f2d1e2e67df
 {{< /tab >}}
 {{% tab name="Files" %}}
-1.  Store the credentials in files with the values encoded in base64:
+1.  Store the credentials in files. The filenames are the keys of the secret:
```

Comment:
```suggestion
1.  Store the credentials in files. The filenames will be the keys of the secret:
```

Reply:
Conflicts with the recommendation by @kbhawkey:
> Possible edit: The filenames are the keys of the secret.

I'm fine either way.

Reply:
Another suggestion:
The filenames represent the keys of the secret?